pandas/tests/reshape/concat/test_concat.py

```diff
@@ -764,14 +766,24 @@ def test_concat_ignore_empty_object_float(empty_dtype, df_dtype):
 def test_concat_ignore_all_na_object_float(empty_dtype, df_dtype):
     df = DataFrame({"foo": [1, 2], "bar": [1, 2]}, dtype=df_dtype)
     empty = DataFrame({"foo": [np.nan], "bar": [np.nan]}, dtype=empty_dtype)
-    result = concat([empty, df], ignore_index=True)
 
     if df_dtype == "int64":
         # TODO what exact behaviour do we want for integer eventually?
         if empty_dtype == "object":
             df_dtype = "object"
         else:
             df_dtype = "float64"
+
+    msg = "The behavior of DataFrame concatenation with all-NA entries"
```

Comment:
Do we have a test for `pd.NA` too?

Reply:
do you mean in instead of np.nan in `empty` on L768?  no.  that would only work for object/None dtypes

Reply:
Ah okay. Cool